WooCommerce/Classes/POS/Utils/Color+WooCommercePOS.swift

```diff
@@ -46,12 +46,30 @@ extension Color {
         )
     }
 
+    static var posOverlayFillInverted: Color {
```

Comment:
If these colors are only used for the success button, I'm wondering if we should keep them within `TotalsView`. WDYT?

Reply:
Oops, alright I think I need new glasses 😂 that looks very funny. As you mention it didn't show before due to the button's background matching the view's background. Addressed on https://github.com/woocommerce/woocommerce-ios/pull/13969/commits/c639cc617fb06321567edef21a28d8189c2e279d by removing the overlay and applying the borders directly to the button's background:

Reply:
(Pressed save too fast )

| - | - |
|--------|--------|
| <img width="518" alt="Screenshot 2024-09-17 at 12 49 25" src="https://github.com/user-attachments/assets/ee0e7f1d-be07-40db-bae4-535d9c545779"> | <img width="518" alt="Screenshot 2024-09-17 at 12 49 41" src="https://github.com/user-attachments/assets/f1d4bf24-2ee7-4598-bf4a-59972f3f5208"> |

> If these colors are only used for the success button, I'm wondering if we should keep them within TotalsView. WDYT?

Agree! Updated on https://github.com/woocommerce/woocommerce-ios/pull/13969/commits/72fdb6fd5bfad864a659068f515d9a92820ffc60